sunaku/home
# yank: add a comment to set allow-passthrough in tmux.conf

- yank: streamline nonzero exits for optional copies
- laptop-display: support DisplayPort; drop nitrogen
- use nvim to run NeoVim and vim to run regular Vim
- chromium isn't stable under any artificial limits
- laptop-display: prefer max resolution as fallback
- ||: POSIX sh(1) does not require a space after ||
- laptop-display: xrandr only honors standalone `-s`
- zsh 5.6.2 isn't rendering strftime %P in $RPROMPT
- zsh prompt: no leading space for single-digit days
- :SudoWrite in Vim failed to show password prompt
- fix putting NodeJS & NPM stuff in ~/opt/install
- add inotifyeval-elixir script to eval files in iex
- avoid string escaping; regex for --sname in prompt
- tmux: restore original layouts under ^B prefix key
- inotify*: recursively watch the current directory
- extract a generic inotifytest script for quick TDD
- zsh: add autopairs plugin for automatic delimiters
- xrandr --size ignored unless standalone afterwards
- zsh: add `k` plugin for `ls`: "k is the new l, yo"
- zsh: forgot to check in *.get file for HSMW plugin
- fontconfig: use Noto Color Emoji with normal fonts
- iex: arrow keys were broken by `tput smkx` setting
- LANG: use standard "en_US.UTF-8" instead of "utf8"
- zsh: add `z` plugin for visited directory frecency
- add configuration for the KiTTY terminal emulator
- gruvbox: blacks were unreadable against background
- alias: `glf` should show ALL fixup commit targets
- tmux: seamless navigation from NeoVim terminals!
- cat-region(1) - print byte offset region from file
- inotifywait-relay(1) - copies close_write over SSH
- aliasing diff to git-diff(1) broke TAB completion
- zathura: use dark colors that are easy on the eyes
- zathura: don't realign to top of page on PgUp/Down
- switch from Tamzen font to "Fira Code" in GUI apps
- tmux: simpler shortcut definitions using variables
- tmux: error binding `M-;` for QWERTY, so use `M-\`
- tmux: faster navigation by running in background
- tmux: only consider #{pane_title} for remote panes
- tmux: simpler logic for detecting NeoVim terminal
- zathura: Space/BackSpace scroll by half a page
- zathura: Alt+Up/Down scroll to top/bottom of page
- inotifywait-relay: preserve timestamp during relay
- tmux: use `-b` option for run-shell instead of `&`
- alias gcm: accept all arguments as commit message
- tmux: detect Vim launched from within `git commit`
- devilspie: maximize and undecorate KiTTY terminals
- tig: revert commits without prompting for editor
- firefox: dark colors for about and extension pages
- terminal: try KiTTY first, then fallback to URxvt
- zathura: square brackets for prev/next page travel
- prompt: bar cursor for EMACS mode, block for VIcmd
- zsh: single tap ESCAPE for VI mode; longer timeout
- prompt: $RPROMPT as standout instead of --NORMAL--
- markdown: cmark now performs `--safe` by default
- prompt: reset cursor to default style in preexec()
- zsh: use add-zsh-hook to register precmd & preexec
- prompt: only use cursor styles for mode indication
- zsh: enable `cdr` for navigating the chdir history
- xstow-facade: create relative symlinks if possible
- switch from XStow to GNU Stow; remodel bin/xstow-*
- stow-facade: be quiet by default; override with -v
- rm-dangling: ask confirmation; pass `-f` to bypass
- tmux-2.9a: set empty status-left instead of length
- switch from Newsbeuter to maintained fork Newsboat
- gemrc: rubygems 2.5.0 renamed --no-document option
- tmux: shell-escape string interpolation with #{q:}
- Revert "zsh: add `k` plugin"; using exa(1) instead
- newsboat: bind the left/right arrow keys like ncdu
- alias: gctf - accept extra args for git-commit(1)
- xterm: enable scrollback buffer for non-tmux usage
- generalize daily(1) into recur(1) with DATE STRING
- prompt: suppress Perforce warnings on $PWD != Root
- devilspie2: add Zathura config; drop ~/.devilspie
- .profile: enable shell history for iex(1) & erl(1)
- fix some bashisms in my usage of echo(1) & grep(1)
- devilspie2: maximize before removing decorations
- README: reproduce text of ISC license for clarity
- README: use actual Unicode emojis in license blurb
- tmux: reduce history-limit to reduce memory usage
- recur: fix zombie apocalypse; support early deaths
- recur: Group ID varies; use pkill -s on Session ID
- recur: setsid auto-forks into background so --wait
- tmux-3.0: swap-window doesn't keep focus on source
- ZSH: bind <C-X><S-R> to run command using sudo(1)
- chromebook: bind VolumeUp/F10 as magic SysRq key!
- new firmware identifies itself as "ZSA Ergodox EZ"
- zsh: expand value of $PAGER in ZLE pager shortcuts
- merge chromebook xkb from GalliumOS into VoidLinux
- switch from Acer C720 to Samsung Chromebook Pro
- kitty: replace clipboard during OSC-52 (no-append)
- prompt: make vcs_info asynchronous using zsh-async
- zsh: some bundles must be loaded AFTER user config
- zsh: skip non-directories to speed up bundle loads
- admit1: eval optional ADMIT1_DENIED envar on denial
- add falcon(1) script to handle `pigeon admit1 ...`
- ps1: also report PGID, SID, TTY; put etime earlier
- kitty: switch from Fira Code font to LigaSrc Pro font
- canary & pigeon: omit command from status messages
- kitty: temporarily disable ligatures at the cursor
- make: don't clear any pre-existing xrdb(1) entries
- alias: add `mov` to move while preserving old name
- xscreensaver obstructs xfce4-screensaver-dialog
- switch from XFCE4 desktop to LXQt (Kvantum themes)
- laptop-display: prefer maximum available resolution
- switch from Samsung Chromebook Pro to Acer Swift 1
- lock screen with slock(1) instead of xscreensaver
- xinitrc: drop syslog viewer and INSERT COIN safety
- kitty: switch from LigaSrc Pro to Cascadia Code PL
- tmux: extract navigation into tmux-navigate plugin
- xinitrc: launch touchpad daemon for palm detection
- fontconfig: set up TrueType interpreter 40 like 35
- tmux: switch from tmux-fingers over to tmux-thumbs
- kitty: support light colorscheme via `make light`
- make: refactor ERB rendering; ~/.themes/background
- kitty: Cascadia Code 2004.30 is larger than before
- prompt: auto-restart vcs_info job: "let it crash!"
- laptop-display: ensure preferred size using --mode
- devilspie2: set kitty "always below" other windows
- kitty: use CascadiaCode-2007.01 SemiLight weight
- kitty: switch to "Liga Fantasque Sans Mono" font
- kitty: use terminal colorscheme from `xrdb -query`
- switch to "gruvbox-material" terminal colorscheme
- SemVer 2.0.0 doesn't prefix "v" on Git tags
- add rmdir-recursive script for lack of `rmdir -r`
- zathura: start in dual view with first page on LHS
- yank: note to set allow-passthrough in tmux.conf
